nuget/helpers/lib/NuGetUpdater/.gitignore

```diff
@@ -2,3 +2,4 @@
 artifacts/
 bin/
 obj/
+Properties/launchSettings.json
```

Comment:
This is a QoL thing to prevent checking in launchSettings that get created when you're trying to run things locally.